sqlmesh/dbt/adapter.py

```diff
@@ -209,6 +216,14 @@ def get_missing_columns(
             if col.name not in target_columns
         ]
 
+    def rename_relation(self, from_relation: BaseRelation, to_relation: BaseRelation) -> None:
+        if from_relation.identifier is not None and to_relation.identifier is not None:
+            new_name = to_relation.identifier
```

Comment:
unnecessary variable

Reply:
Can new_name be in a different schema or database? This implementation assumes its the same.

Reply:
> Can new_name be in a different schema or database? This implementation assumes its the same.

Provided that this operation results in an `ALTER TABLE table RENAME TO table_name`, I assumed just the identifier would suffice.

Reply:
> unnecessary variable

Removed.